instrumentation/kamon-elasticsearch/src/main/scala/kamon/instrumentation/elasticsearch/RequestNameConverter.scala

```diff
@@ -0,0 +1,12 @@
+package kamon.instrumentation.elasticsearch
+
+object RequestNameConverter {
+  def convert(className: String, fallback: String): String = {
+    // converts class name to something more appropriate
+    // e.g. ListTasksRequest to list.tasks.requests
+    Option(className)
+      .getOrElse(fallback)
+      .replaceAll("([^_A-Z])([A-Z])", "$1.$2")
+      .toLowerCase
```

Comment:
Few things that would be better here:
  - Add the `elasticsearch` prefix to all names.
  - Drop the `Request` at the end of the name.
  - Cache these transformed operation names. The number of possible options is bound so it wouldn't be a problem.